api/v1alpha1/contentlibrary_types.go

```diff
@@ -44,6 +44,20 @@ type StorageBacking struct {
 	DatastoreID string `json:"datastoreID,omitempty"`
 }
 
+// ResourceNamingStrategy represents a naming strategy for item resources in a content library in vCenter.
+type ResourceNamingStrategy string
+
+const (
+	// ResourceNamingStrategyFromItemID indicates the naming strategy that generates the item resource name from the item
+	// identifier for items in a content library.
+	ResourceNamingStrategyFromItemID ResourceNamingStrategy = "FROM_ITEM_ID"
```

Comment:
Maybe indicate this is the default?

Reply:
It is commented in the ResourceNamingStrategy field on the library specs, can add a comment here too.